src/components/DayPicker.jsx

```diff
@@ -55,6 +55,7 @@ const propTypes = forbidExtraProps({
   hideKeyboardShortcutsPanel: PropTypes.bool,
   daySize: nonNegativeInteger,
   isRTL: PropTypes.bool,
+  verticalHeight: PropTypes.number,
```

Comment:
should this be a nonNegativeInteger?

Reply:
sounds good, I can't think of a case where it would in fact make sense to use a negative height 🤔